Terraform crash on version terraform-provider-scaleway 1.12.0

Hi there,
I tried creating an instance volume and instance server using scaleway provider, but it crashes. I'm pasting relevant crash message and I'm available for further investigation.

### Terraform Version

`0.12.16`

### Affected Resource(s)

- `scaleway_instance_volume`
- `scaleway_instance_server`

Possibly others involving scaleway provider.

### Terraform Configuration Files

```
variable "region" {}
variable "datacenter" {}
variable "scaleway_access_key" {}
variable "scaleway_secret_key" {}
variable "scaleway_organization_id" {}

provider "scaleway" {
  access_key      = var.scaleway_access_key
  secret_key      = var.scaleway_secret_key
  organization_id = var.scaleway_organization_id
  zone            = var.datacenter
  region          = var.region
}

resource "scaleway_instance_volume" "a" {
  type = "b_ssd"
  count = "1"
  name  = "name"
  size_in_gb = 30
}
```

### Debug Output

```
panic: runtime error: invalid memory address or nil pointer dereference
2019-11-25T23:46:56.271Z [DEBUG] plugin.terraform-provider-scaleway_v1.12.0_x4: [signal SIGSEGV: segmentation violation code=0x1 addr=0x58 pc=0xde3f2e]
2019-11-25T23:46:56.271Z [DEBUG] plugin.terraform-provider-scaleway_v1.12.0_x4: 
2019-11-25T23:46:56.271Z [DEBUG] plugin.terraform-provider-scaleway_v1.12.0_x4: goroutine 13 [running]:
2019-11-25T23:46:56.271Z [DEBUG] plugin.terraform-provider-scaleway_v1.12.0_x4: github.com/nicolai86/scaleway-sdk.(*API).GetServerAvailabilities(0x0, 0x0, 0x0, 0x0)
2019-11-25T23:46:56.271Z [DEBUG] plugin.terraform-provider-scaleway_v1.12.0_x4: 	/opt/teamcity-agent/work/5d79fe75d4460a2f/pkg/mod/github.com/nicolai86/scaleway-sdk@v0.0.0-20181024210327-b20018e944c4/availability.go:36 +0x4e
2019-11-25T23:46:56.271Z [DEBUG] plugin.terraform-provider-scaleway_v1.12.0_x4: github.com/terraform-providers/terraform-provider-scaleway/scaleway.providerConfigure(0xc00010ab60, 0x0, 0xc000556980, 0xc00010ab60, 0x0)
2019-11-25T23:46:56.271Z [DEBUG] plugin.terraform-provider-scaleway_v1.12.0_x4: 	/opt/teamcity-agent/work/5d79fe75d4460a2f/src/github.com/terraform-providers/terraform-provider-scaleway/scaleway/provider.go:285 +0x383
2019-11-25T23:46:56.271Z [DEBUG] plugin.terraform-provider-scaleway_v1.12.0_x4: github.com/hashicorp/terraform/helper/schema.(*Provider).Configure(0xc000464100, 0xc000231e60, 0x101d3c0, 0xc000231dd0)
2019-11-25T23:46:56.271Z [DEBUG] plugin.terraform-provider-scaleway_v1.12.0_x4: 	/opt/teamcity-agent/work/5d79fe75d4460a2f/pkg/mod/github.com/hashicorp/terraform@v0.12.8/helper/schema/provider.go:270 +0xd2
2019-11-25T23:46:56.271Z [DEBUG] plugin.terraform-provider-scaleway_v1.12.0_x4: github.com/hashicorp/terraform/helper/plugin.(*GRPCProviderServer).Configure(0xc0000acb20, 0x142c800, 0xc0002319b0, 0xc00022fdc0, 0xc0000acb20, 0xc0002319b0, 0xc000502a80)
```

### Expected Behavior

Terraform should plan creation of resources.

### Actual Behavior

Terraform crashes upon planning.

### Steps to Reproduce

terraform plan

--- a/scaleway/provider.go
+++ b/scaleway/provider.go
@@ -288,7 +288,7 @@ func providerConfigure(d *schema.ResourceData, terraformVersion string) (interfa
 		TerraformVersion:      terraformVersion,
 	}
 
-	meta.bootstrap()
+	err = meta.bootstrap()
 	if err != nil {
 		return nil, err
 	}

